AsyncDisplayKit/ASCollectionNode.mm

```diff
@@ -219,6 +219,11 @@ - (void)setTuningParameters:(ASRangeTuningParameters)tuningParameters forRangeMo
   return [self.view.rangeController setTuningParameters:tuningParameters forRangeMode:rangeMode rangeType:rangeType];
 }
 
+- (void)updateCurrentRangeWithMode:(ASLayoutRangeMode)rangeMode;
+{
+  [self.view.rangeController updateCurrentRangeWithMode:rangeMode];
```

Comment:
We should be careful not to trigger view creation...I might say we should entirely skip / ignore the call if someone calls this before self.isNodeLoaded.